deploy/injector-deployment.yaml

```diff
@@ -76,6 +76,14 @@ spec:
               value: "vault-agent-injector-svc,vault-agent-injector-svc.$(NAMESPACE),vault-agent-injector-svc.$(NAMESPACE).svc"
             - name: AGENT_INJECT_USE_LEADER_ELECTOR
               value: "true"
+            - name: AGENT_INJECT_CPU_REQUEST
```

Comment:
Would you mind setting these values to the backend defaults? These defaults have been here for a while and don't want to change the existing behavior.

```
	DefaultResourceLimitCPU   = "500m"
	DefaultResourceLimitMem   = "128Mi"
	DefaultResourceRequestCPU = "250m"
	DefaultResourceRequestMem = "64Mi"
```

Reply:
oh my mistake, that was my intent actually, but I was looking at the values in one of the unit tests and didn't realize they were the non-defaults. Updating now!

Reply:
@jasonodonnell done, updated